app/views/form-designer/edit-answer-type.html

```diff
@@ -0,0 +1,142 @@
+{% extends "layout-govuk-forms.html" %}
+
+{% set pageTitle -%}
+  {% if pageData['long-title'] %}
+    {{pageData['long-title']}}
+  {% else %}
+    Edit question
+  {% endif %}
+{%- endset %}
+
+{% block pageTitle %}
+  {{pageTitle|safe}} - GOV.UK Forms
+{% endblock %}
+
+{% block beforeContent %}
+  {% set prevPageId = pageId | int - 1 %}
+  {% if prevPageId > 0 %}
+    <a class="govuk-back-link" href="{{prevPageId}}">Back</a>
```

Comment:
Not sure this is the behaviour I'd expect - this will go to the answer type page for the previous question (i.e. if you're editing page 5, this will take you to `/form-designer/edit-answer-type/4`. I think I'd always expect this back link to go back to the question page (i.e. `/form-designer/edit-page/5`.

Reply:
Good spot! I've taken a look and think I have something working. 

Back should now work closer to intended.

Reply:
Nice one!